plugins/chat/spec/system/new_message_spec.rb

```diff
@@ -53,6 +53,31 @@
   end
 
   context "with no selection" do
+    context "with unread state" do
+      fab!(:user_1) { Fabricate(:user) }
+      fab!(:channel_1) { Fabricate(:chat_channel) }
+      fab!(:channel_2) { Fabricate(:direct_message_channel, users: [current_user, user_1]) }
+
+      before do
+        channel_1.add(user_1)
+        channel_1.add(current_user)
+        Fabricate(:chat_message, chat_channel: channel_1, user: user_1)
+        Fabricate(:chat_message, chat_channel: channel_2, user: user_1)
+      end
+
+      it "shows it" do
```

Comment:
I guess it works hehe

Reply:
yes right I can do better here 😅  will change :+1: